### DISCUSSION TITLE
Lnurl and phoenixd

### DISCUSSION THREAD

**Discussion by Alex71btc** [ACINQ/phoenixd#44] [General]
When using the new lnbits integration by bitcarrot https://github.com/bitkarrot/lnbits-phoenixd the Lnurlp extension in lnbits does not work. I guess this is due to phoenixd inability to process  Lnurl strings?

**Comment by Bashy** [comment]
As i described [here](https://github.com/lnbits/lnbits/issues/2478#issuecomment-2106042401), it is possible to make lnbits/lnurlp works, but there might be some undesirable side effects, I haven't deeply tested it.

As I understand, `phoenixd` currently does not support lnurl `description_hash/unhashed_description`, there is thus not any lnurl metadata verification/correlation made by `phoenixd` at the moment of payment request (invoice) creation

**Comment by Alex71btc** [comment]
LNURLp in Lnbits still not working after new release today , getting still Error about description hash...
![grafik](https://github.com/ACINQ/phoenixd/assets/105175715/1599422f-c101-45de-88f7-bdf1ffc2ec1e)

**Comment by Alex71btc** [comment]
Works now with latest dev branch of lnbits.Soon will be in release I hope.
